app/models/unavailability.rb

```diff
@@ -6,4 +6,13 @@ class Unavailability < ActiveRecord::Base
   # this will serialize the columns of the unavailability model
   # so that they accept strings in the form of arrays
   Date::DAYNAMES.map(&:downcase).map(&:to_sym).each { |d| serialize d, Array }
+
+  def grid
```

Comment:
I know I wrote this, but a comment explaining what type of data structure this method returns could be useful.